docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,88 @@
+
+The panel can then be used as a data source for other panels, by using the inbuilt `-- Dashboard --` data source to pull the datagrid data. This allows for an interactive dashboard experience, where you can modify the data and see the changes reflected in other panels.
+
+To get a better understanding of the inbuilt `-- Grafana --` and `-- Dashboard --` datasources, please check the [special data sources](/docs/grafana/latest/datasources/#special-data-sources) documentation.
+
+## Selecting series
+
+If multiple series are present, the datagrid can display the preferred dataset through the panel options `Select series` select box.
+
+## How to use the datagrid
+
+Datagrid offers various ways of interacting with the data. You can add, edit, move, clear, and remove rows and columns, as well as use the inbuilt search functionality to find specific data, convert field types, or freeze horizontal scroll on a specific column.
+
+### Adding data
+
+Data can be added to the datagrid by creating a new column or row. To create a new column, click the `+` button found after the last column. After clicking the button, a new input will prompt you to name the newly created column. Clicking anywhere outside the input, or pressing the `Enter` key, will finalize column creation. After that, data can be filled on each cell.
+
+To add a new row, click the `+` button found after the last row. The button is present on each cell after the last row and will trigger creating a new one while also activating the corresponding cell that was clicked.
+
+### Editing data
+
+Data can be edited by double clicking on the cell that needs to be modified. This will activate the cell and allow you to edit the data. After editing the data, clicking anywhere outside the cell or pressing the `Enter` key will finalize the edit. To easily clear a cell of data, you can click on a cell to focus it and then press the `Delete` key.
+
+### Moving data
+
+Columns and rows can be moved around as desired. To move a column, click and hold the column header and drag it to the desired location. Releasing the mouse button will finalize the move. To move a row, click and hold the row selector from the number column situated on the left side and drag it to the desired location. Releasing the mouse button will finalize the move.
+
+### Selecting multiple cells
+
+You can select multiple cells by clicking on a single cell and dragging the mouse across others. This selection can be used to copy the data from the selected cells or to delete them using the `Delete` key.
+
+### Using the context menu
```

Comment:
This section is more descriptive of a UI element than really directions on how to use a context menu—folks know how to use a context menu. I would move this up to just before the use case section and make it an H2. I've also suggested moving the header menu content under this since they're both menus.
```suggestion
### Context menu
```